components/brave_rewards/resources/ui/utils.ts

```diff
@@ -125,3 +125,11 @@ export const donationTotal = (report: Rewards.Report) => {
   const tips = new BigNumber(report.tips)
   return new BigNumber(report.donation).plus(tips).dividedBy('1e18').toFixed(1, BigNumber.ROUND_DOWN)
 }
+
+export const constructBackupString = (backupKey: string) => {
+  return 'Brave Wallet Recovery Key<BR>Date created: ' + new Date(Date.now()).toLocaleDateString() + '<BR><BR>Recovery Key: ' + backupKey +
```

Comment:
i also think you want newlines instead of `<br>` because this is being saved as a plaintext file, not HTML. https://github.com/brave/brave-core/pull/816/files#diff-aa81b445f3d957d5c028d1df07d3ac64R101

Reply:
I do, and I tried with that but it wasn't breaking the lines. I'll see what I can do with it.

Reply:
interesting. if i open devtools and enter `document.body.innerText = 'hello \n\n world'`, it seems to show newlines as expected.

Reply:
Done.